Formula/tepl.rb

```diff
@@ -71,7 +79,7 @@ def install
       -I#{gtksourceview4.opt_include}/gtksourceview-4
       -I#{gtkx3.opt_include}/gtk-3.0
       -I#{harfbuzz.opt_include}/harfbuzz
-      -I#{include}/tepl-4
+      -I#{include}/tepl-5
```

Comment:
This should work right @SeekingMeaning ?
```suggestion
      -I#{include}/tepl-#{version.major}
```

Reply:
Yes